src/uct/ib/base/ib_md.c

```diff
@@ -801,6 +801,9 @@ static ucs_status_t uct_ib_mem_reg(uct_md_h uct_md, void *address, size_t length
 
     memh = uct_ib_memh_alloc(md);
     if (memh == NULL) {
+        ucs_log((flags & UCT_MD_MEM_FLAG_HIDE_ERRORS) ? UCS_LOG_LEVEL_DIAG :
```

Comment:
maybe uct_md_reg_log_lvl should be helper in uct_md.h, reuse it here?

Reply:
ok, also thought about this